build/media-suite_compile.sh

```diff
@@ -1359,23 +1359,27 @@ if [[ ! $x265 = "n" ]] && do_vcs "hg::https://bitbucket.org/multicoreware/x265";
     do_uninstall libx265{_main10,_main12}.a bin-video/libx265_main{10,12}.dll "${_check[@]}"
     [[ $bits = "32bit" ]] && assembly="-DENABLE_ASSEMBLY=OFF"
     [[ $x265 = d ]] && xpsupport="-DWINXP_SUPPORT=ON"
+    do_patch "https://gist.githubusercontent.com/1480c1/7743056f52e4a546294e2160431674bc/raw/0001-CMake-alias-CMAKE_INSTALL_BINDIR-to-BIN_INSTALL_DIR.patch"
```

Comment:
Did you submit it upstream?

Reply:
Not entirely sure how to do pr and stuff with hg and bitbucket. I want to, but still need to learn how to use those

Reply:
Sure, as long as you're thinking of doing it. Unless a project is dead or the patch would be considered unmergeable (like FFmpeg's configure patch), it's better to consider them temporary.

Reply:
Right now reading their CLA and trying to figure out how to use patchwork

Reply:
Don't forget to check whether they're using GNUInstallDirs already. If not, then the patch(es) would need to be more than just using CMAKE_INSTALL_BINDIR, or it could be refused.

Reply:
I think they weren't using GNUInstallDirs, that's why I added in the patch
``` patch
+if(NOT CMAKE_INSTALL_BINDIR)
+    set(CMAKE_INSTALL_BINDIR bin)
+endif()
```

Reply:
My main problem is that since they have a minimum of 2.8.8 for cmake, I need to make sure whatever I write supports 2.8.8, but the cmake docs online only go to 3.0.2

Reply:
You could probably propose updating the minimum version too. How old is 3.0.2 already? There's absolutely no reason why versions older than one or two years should be supported (or the latest version in an Ubuntu LTS usually).

Reply:
Even old-ass xenial (Ubuntu 16.04 LTS) has 3.5.1. Propose an update to that version, at the very least.

Reply:
I will try, 3.5 is at least a pretty good version to start with

Reply:
It seems centos 7 still uses 2.8.12, holy hell their packages are out of date

Reply:
If you're building stuff from hg/git you can also not use a CMake from 10 years ago. Aim for 3.5.1, at least.

Reply:
Probably best to forget that centos 7 exists